content/docs/5.1/develop/clients/java.md

```diff
@@ -73,9 +73,9 @@ for(Kvrpcpb.KvPair pair : list) {
 }
 ```
 
-These functions also have batch variants (`batchPut`, `batchGet`, `batchDelete` and `batchScan`), which offer considerably reduced network overhead and can result in dramatic performance increases under certain workloads.
+These functions also have batch variants (`batchPut`, `batchGet`, `batchDelete`, and `batchScan`), which considerably reduce network overhead and increase performance under certain workloads.
```

Comment:
```suggestion
These functions also have batch variants (`batchPut`, `batchGet`, `batchDelete`, and `batchScan`), which considerably reduces network overhead and increases performance under certain workloads.
```

Reply:
Here I think `which` means `batch variants` and it is in plural form.

Reply:
Then please remove the comma before `which`.

Reply:
Done. PTAL~